src/magma/vk/textures/CustomImageView.cpp

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file CustomImageView.cpp
+ * @brief Class contain custom image view
```

Comment:
Typo: contain -> containing

if it's *.cpp file, I guess there should be some different info.
Maybe like:
```
Implementation of CustomImageView methods
```

Reply:
Done